config/landing_pages/e/phptek.yml

```
@@ -0,0 +1,231 @@
+            </div>
+
+  - row:
+    - column:
+        - type: line_divider
+
+  - row:
+    - column:
+        - type: header
+          header:
+            title:
+              text: "Nexmo ❤️ PHP"
+              align: center
+            icon:
+              name: "icon-code"
+              color: "blue-dark"
+        - type: text
+          text:
+            content: "At Nexmo, we love PHP - it's one of our most-used and most-loved languages by customers and Developer Advocates alike. Start your Nexmo project by installing the Nexmo PHP library:"
+        - type: text
+          text:
+            content: |
+              ```
+              composer require nexmo/client
+              ```
+        - type: text
+          text:
+            content: "Or if Laravel's your thing, we have a prebuilt package that wraps the Nexmo client, reads your config from `.env` and makes it available through the `Nexmo` facade"
+        - type: text
+          text:
+            content: |
+              ```
+              composer require nexmo/laravel
+              ```
+
+    - column:
+        - type: header
+          header:
+            title:
+              text: "Get started by making a call"
+              align: center
+            icon:
+              name: "icon-call"
+              color: "blue-dark"
+        - type: text
+          text:
+            content: |
+              ```php
+              $call = $client->calls()->create([
+                  'to' => [[
+                      'type' => 'phone',
+                      'number' => TO_NUMBER
+                  ]],
+                  'from' => [
+                      'type' => 'phone',
+                      'number' => NEXMO_NUMBER
+                  ],
+                  'ncco' => [
+                      [
+                          'action' => 'talk',
+                          'text' => 'This is a text to speech call from Nexmo'
+                      ]
+                  ]
+              ]);
+              ```
+              <div class="Vlt-right">
+              <a class="Vlt-btn Vlt-btn--secondary" href="/voice/voice-api/code-snippets/make-an-outbound-call-with-ncco/php">View the complete code &raquo;</a>
+              </div>
+
+  - row:
+    - column:
+        - type: line_divider
+
+  - row:
+    - column:
+      - type: section_header
+        section_header:
+          title: "Handy PHP-related links and further reading"
+          icon:
+            name: "icon-pin-2"
+            color: "blue-dark"
+
+
+  - row:
+    - column:
+      - type: text
+        text:
+          content: >
+            <div class="Vlt-callout Vlt-callout--warning">
+            	<i></i>
+            	<div class="Vlt-callout__content">
+                <h4>Can't find what you're looking for?</h4>
+                Come and say hello at the Nexmo booth! We'd love to hear what you're building, and if anything trips you then we'd love to help you out (and improve the docs at the same time!)
```

Suggested rewording for "trips you" and using "love" twice in consecutive sentences:

```suggestion
Come and say hello at the Nexmo booth! We'd love to hear what you're building with our APIs. If you're struggling with something then we'll help you out (and improve our docs at the same time!)
```

---

https://github.com/Nexmo/nexmo-developer/pull/1706#discussion_r285608266